src-built-in/components/toolbar/components/Search.jsx

```diff
@@ -106,7 +106,7 @@ export default class Search extends React.Component {
 	componentDidUpdate() {
 		if (this.state.hotkeySet) {
 			FSBL.Clients.WindowClient.finWindow.focus(() => {
-				this.searchContainer.current.focus();
+				this.searchContainer.current.querySelector('#searchInput').focus()
```

Comment:
Is there a way to follow Reacts way of accessing components vs. selecting dom elements directly? Maybe it's really hard in the current framework...